src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -283,3 +283,14 @@ interfaces:
     field_name_patterns:
       name: book_2
     timeout_millis: 30000
+  - name: StreamShelves
+    request_object_method: false
+    page_streaming:
```

Comment:
I'm fine with them not being mutually exclusive. I just wish that the names of the two features were more distinct (ie didn't both have the word "streaming" in them). Like ("pagination", "streaming") rather than ("page streaming", "streaming"). But that's a non-trivial change at this point (and out of scope for this CL).